.chloggen/module-info-builder.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: cmd/builder
```

```suggestion
component: cmd/builder, otelcol
```
Maybe this should just be a separate changelog entry, but we have two separate changes in here:
1. Adding support in the builder for baking this information into the binary
2. Adding support in the `components` subcommand for printing this information.

---

Yeah, good call. I'll push a separate changelog entry